packages/react/src/components/Skeleton/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
This should be updated in the WSO2 ESLint Plugin.

cc:/ @savindi7